recipes/sqlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="3.11.1" %}
+{% set version_split=version.split(".") %}
+{% set major=version_split[0] %}
+{% set minor=version_split[1]|int * 100 %}
+{% set bugfix=version_split[2]|int * 100 %}
+{% set version_coded=(major ~ (minor|string)[:3] ~ (bugfix|string)[:3]) %}
```

Comment:
This is just way too complex - given the SHA changes, I'm not too worried about having redundancy in the recipe.